configure.sh

```diff
@@ -163,20 +188,17 @@ config_host_linux()
             warn "libseccomp < 2.4.1 has known vulnerabilities"
             warn "Proceeding anyway, but consider upgrading"
         fi
-        MAKECONF_SPT_CFLAGS=$(pkg-config --cflags libseccomp)
-        MAKECONF_SPT_LDLIBS=$(pkg-config --libs libseccomp)
+        MAKECONF_LIBSECCOMP_CFLAGS="$(pkg-config --cflags libseccomp)"
+        MAKECONF_LIBSECCOMP_LDLIBS="$(pkg-config --libs libseccomp)"
     fi
-    if ! PKG_CFLAGS=${MAKECONF_SPT_CFLAGS} gcc_check_header seccomp.h; then
+    if ! PKG_CFLAGS="${MAKECONF_LIBSECCOMP_CFLAGS}" gcc_check_header seccomp.h; then
         die "Could not compile with seccomp.h"
     fi
-    if ! PKG_LIBS=${MAKECONF_SPT_LIBS} gcc_check_lib -lseccomp; then
-        die "Could not link with -lseccomp"
+    if [ -n "${MAKECONF_LIBSECCOMP_LDLIBS}" ]; then
+        if ! gcc_check_lib "${MAKECONF_LIBSECCOMP_LDLIBS}"; then
```

Comment:
This check doesn't work on my computer / in my CI testing. 
Removing the quotes seems to do the trick but I'm not sure of the implications.
```suggestion
        if ! gcc_check_lib ${MAKECONF_LIBSECCOMP_LDLIBS}; then
```

Reply:
Thanks, yes, the quoting is a bit subtle there, I think `$MAKECONF_LIBSECCOMP_LDLIBS` should get passed as multiple arguments. I really should also run all the shell code through shellcheck.

I've made that change, rebased on master and force-pushed.

For reference, what is the value of that variable / the output of `pkg-config --libs libseccomp` on your machine? And what Linux distro are you on?

Reply:
Okay thank you, this works for me. 

On my machine the output is `-lseccomp`, using Archlinux.